bin/action_subscriber

```diff
@@ -39,23 +40,16 @@ module ActionSubscriber
     end
   end
 
-  [:INT, :QUIT, :TERM].each do |signal|
-    trap(signal) do
-      puts "Stopping server..."
-      wait_loops = 0
+  if ::RUBY_PLATFORM == "java"
```

Comment:
if I just use `at_exit` then MRI throws an error about "`sleep': Interrupt"
* this doesn't actually cause an error, but seemed like it would be alarming to most users. I tried several workarounds like trying to wait for the sleep to finish and exiting the main thread of execution before I finished shutting down, but I couldn't get it to work cleanly

using the `trap(signal)` under jRuby causes it to hang during shutdown.
* I think this is because trapping these signals overrides some JVM stuff that is used for its own graceful shutdown